testing/src/test/java/org/oppia/android/testing/robolectric/OppiaShadowActivityManagerTest.kt

```diff
@@ -32,6 +32,15 @@ private val oppiaShadowActivityManager by lazy { OppiaShadowActivityManager() }
   shadows = [OppiaShadowActivityManager::class]
 )
 class OppiaShadowActivityManagerTest {
+  @Inject
+  lateinit var context: Context
+
+  private val oppiaShadowActivityManager: OppiaShadowActivityManager by lazy {
+    shadowOf(
+      context.getSystemService(Context.ACTIVITY_SERVICE) as ActivityManager
+    ) as OppiaShadowActivityManager
+  }
```

Comment:
A small nit, but we should do this like a val getter since it's considered an antipattern to have long-term references to shadows. We can make this uncached by doing something like:

```kotlin
private val oppiaShadowActivityManager: OppiaShadowActivityManager
  get() = shadowOf(...) as OppiaShadowActivityManager
```

(Or ``get() {}`` for multi-line cases).

Reply:
done.